client/templates/timeout.tmpl

```diff
@@ -70,3 +118,54 @@ func (c * {{$decorator}}) {{$method.Declaration}} {
     {{$method.Pass ("c.client.") }}
 }
 {{end}}
+{{end}}
+
+{{/* ↓↓↓ Special Case APIs ↓↓↓ */}}
```

Comment:
I suggest moving special case APIs out of generated code.

Reply:
Done. I left it in the current file